app/assets/javascripts/discourse/app/templates/tags/show.hbs

```diff
@@ -34,6 +34,13 @@
             }}
           {{/unless}}
         {{/if}}
+        
+        {{plugin-outlet name="before-create-topic-button"
+        args=(hash
```

Comment:
This is super minor but the indentation here is off. Could you line up `args` with `plugin-outlet`?

Reply:
Sure, thanks, I did that.

Reply:
Great, we've tagged it for merging after the next release.